.dotnet/examples/OpenAI.Examples.csproj

```diff
@@ -1,8 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-    <!--Ignore XML doc comments on test types and members-->
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <!-- CS1591 - Ignore XML doc comments on test types and members-->
+    <!-- OPENAI001 - Ignore experimental warnings-->
+    <NoWarn>$(NoWarn);CS1591;OPENAI001;</NoWarn>
```

Comment:
nit: Could you split this into two separate `NoWarn` like you did in the OpenAI.csproj?